tests/layer_tests/tensorflow_tests/test_tf_ConjugateTranspose.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+import numpy as np
+from common.tf_layer_test_class import CommonTFLayerTest
+
+# Testing operation ConjugateTranspose
+# Documentation: https://www.tensorflow.org/api_docs/python/tf/raw_ops/ConjugateTranspose
+
+
+class TestConjugateTranspose(CommonTFLayerTest):
+    
+    def _prepare_input(self, inputs_info):
+        
+        assert 'x' in inputs_info
+        input_shape = inputs_info['x']
+        inputs_data = {}
+        inputs_data['x'] = np.random.default_rng().random(input_shape).astype(np.complex64)
+        return inputs_data
+    
+    def create_conjugate_transpose_net(self, shape, perm):
+        """
+            TensorFlow net                                 IR net
+
+            Placeholder->ConjugateTranspose    =>       Placeholder->Transpose->Conjugate->Transpose
+        """
+
+        import tensorflow as tf
+
+        tf.compat.v1.reset_default_graph()
+
+        # Create the graph and model
+        with tf.compat.v1.Session() as sess:
+            tf_input = tf.compat.v1.placeholder(tf.complex64, shape, "Input")
+
+            tf.raw_ops.ConjugateTranspose(x=tf_input, perm=perm, name = "Operation")
```

Comment:
not quite correct, please hide complex type to intermediate layers because we don't support complex type for Parameters and Result nodes. We just imitate this support by expressing a tensor of complex type as floating-point tensor with auxiliary dimension.
Example of such layer test for operation with complex type you can see here: https://github.com/openvinotoolkit/openvino/blob/master/tests/layer_tests/tensorflow_tests/test_tf_ComplexFFT.py.
You should pass real and imag parts from outside.